client/ayon_blender/api/lib.py

```diff
@@ -252,19 +252,23 @@ def lsattrs(attrs: Dict) -> List:
             continue
         for node in getattr(bpy.data, coll):
             for attr, value in attrs.items():
-                avalon_prop = node.get(pipeline.AVALON_PROPERTY)
-                if not avalon_prop:
+                ayon_prop = node.get(pipeline.AYON_PROPERTY) or (
+                    node.get(pipeline.AVALON_PROPERTY)
+                )
+                if not ayon_prop:
                     continue
-                if (avalon_prop.get(attr)
-                        and (value is None or avalon_prop.get(attr) == value)):
+                if (ayon_prop.get(attr)
+                        and (value is None or ayon_prop.get(attr) == value)):
                     matches.add(node)
     return list(matches)
 
 
 def read(node: bpy.types.bpy_struct_meta_idprop):
     """Return user-defined attributes from `node`"""
 
-    data = dict(node.get(pipeline.AVALON_PROPERTY, {}))
+    data = dict(node.get(pipeline.AYON_PROPERTY, {}))
+    if node.get(pipeline.AVALON_PROPERTY, {}):
+        data.update(dict(node.get(pipeline.AVALON_PROPERTY, {})))
```

Comment:
This would **always** make any avalon property override any ayon property.
I'd say, if we want that - we should do the reverse and have AYON override the legacy one.
But better yet most likely is to only get the AVALON property data if there is no AYON property data.